codespell_lib/data/dictionary.txt

```diff
@@ -6023,6 +6038,7 @@ chipslect->chipselect
 chipstes->chipsets
 chiuldren->children
 chked->checked
+chlids->children
```

Comment:
Possibly this too (as a variable/function), but I suspect it might upset our CI...
```suggestion
chlids->children, childs,
```

Reply:
It seems like it will, so I think we'll have to skip this one.